app/assets/javascripts/discourse/app/deprecation-workflow.js

```diff
@@ -7,10 +7,10 @@ const DEPRECATION_WORKFLOW = [
     matchId: "discourse.decorate-widget.hamburger-widget-links",
   },
   // TODO (glimmer-post-stream): remove the silence once upgrade notes are ready and we start rolling out the new version
-  {
-    handler: "silence",
-    matchId: "discourse.post-stream-widget-overrides",
-  },
+  // {
+  //   handler: "silence",
+  //   matchId: "discourse.post-stream-widget-overrides",
+  // },
```

Comment:
I guess we should silence again for now? And then unsilence once upgrade docs are published?

```suggestion
  {
    handler: "silence",
    matchId: "discourse.post-stream-widget-overrides",
  },
```